pkg/koordlet/util/system/resctrl.go

```diff
@@ -90,6 +97,15 @@ func isKernelSupportResctrl() (bool, error) {
 	return isCatFlagSet && isMbaFlagSet, nil
 }
 
+func IsVendorSupportResctrl() bool {
+	vendorID, err := GetVendorIDByCPUInfo(GetCPUInfoPath())
+	if err != nil {
+		klog.V(4).Infof("GetVendorIDByCPUInfo error: %v", err)
+		return false
+	}
+	return vendorID == AMD_VENDOR_ID || vendorID == INTEL_VENDOR_ID
```

Comment:
ARM can support resctrl via MPAM.

Reply:
I plan to add support for ARM architecture in future prs.

Reply:
> OK. We can leave a TODO. However, please do not fail for the ARM platform since we did support it.

Since the ARM platform has numerous [vendor IDs](https://github.com/util-linux/util-linux/blob/0284eb3a8a6505dd9745b042089046ad368bfe74/sys-utils/lscpu-arm.c#L294-L315), I included an ARM vendor map to verify vendor support for various CPUs.